### ISSUE TITLE
ovms pipeline run on walserv3 failed

### ISSUE THREAD

**Issue by vli11** [intel-retail/automated-self-checkout#135]
## Version

What release version was the issue found in? 1.5.0

## Hardware configuration

- Intel® xeon Platinum 8351N with Intel® Data Center GPU Flex 140

## Describe the bug
command to run:
 ./docker-run.sh --workload opencv-ovms --platform dgpu.1 --inputsrc http://127.0.0.1:8554/camera_0

here is the failure error from results/maskrcnn.log
Start processing:
        Model name: instance-segmentation-security-1040
Traceback (most recent call last):
  File "/scripts/grpc_infer_binary_maskrcnn-omz.py", line 154, in <module>
    response = grpc_stub.ModelInfer(request)
  File "/usr/local/lib/python3.8/dist-packages/grpc/_channel.py", line 1030, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/usr/local/lib/python3.8/dist-packages/grpc/_channel.py", line 910, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
        status = StatusCode.NOT_FOUND
        details = "Model with requested version is not found"
        debug_error_string = "UNKNOWN:Error received from peer  {created_time:"2023-06-22T20:05:58.958911509+00:00", grpc_status:5, grpc_message:"Model with requested version is not found"}"
>

## To Reproduce

1. login to walserv3
2. make build-ovms-server
3. make build-ovms-client
4. update configs/opencv-ovms/models/2022/config.json device from  "CPU" to "GPU.1"
5.  ./docker-run.sh --workload opencv-ovms --platform dgpu.1 --inputsrc http://127.0.0.1:8554/camera_0
6. view the log file results/maskrcnn.log, it should not have any error

**Comment by jim-wang-intel** [comment]
@vli11   should run it with `sudo` for docker-run.sh command as it is the suggested way

**Comment by vli11** [comment]
The code has been updated, we don't use maskcnn.log anymore. Current log becomes  r0.jsonl and pipeline0.log files. I have tested the bit model case in walserv3:
1. updated configs/opencv-ovms/cmd_client/res/grpc_python/configuration.yaml to use model bit_64
2. updated configs/opencv-ovms/models/2022/config.json the "target_device" from "CPU" to "GPU.1"
3. run with `PIPELINE_PROFILE="grpc_python" sudo -E ./docker-run.sh --workload opencv-ovms --platform dgpu.1 --inputsrc rtsp://127.0.0.1:8554/camera_0`

The pipeline was run successfully without error, it produced correct r0.jsonl and pipeline0.log. closed this issue since issue can not be produced anymore
